frontend/src/index.pug

```diff
@@ -37,10 +37,15 @@ html
           .timestamp-footer
             span Generated by &nbsp;
             a(
-              v-bind:href="getRepoSenseLink()", target="_blank",
+              v-bind:href="getRepoSenseHomeLink()", target="_blank",
             )
-              strong RepoSense {{ repoSenseVersion }}
-            span &nbsp; on {{ creationDate }}
+              strong RepoSense
+            span &nbsp;(
+            a(
+              v-bind:href="getRepoSenseVersionLink()", target ="_blank",
+            )
+              strong {{ repoSenseVersion }}
+            span ) on {{ creationDate }}
```

Comment:
you didn't, hence my comment earlier.

![image](https://user-images.githubusercontent.com/19277206/65525865-bcbe9700-df22-11e9-9601-0933f4340fa9.png)

Double check your PR;that's the only area with 2 spaces

Reply:
Oh okay. Got it now. My bad. The  ** &nbsp;** was causing two spaces. Have fixed it. Thanks @eugenepeh for guiding.